satellite/dbcleanup/chore.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbcleanup
+
+import (
+	"context"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/satellite/orders"
+)
+
+var (
+	// Error the default dbcleanup errs class
+	Error = errs.Class("dbcleanup error")
+
+	mon = monkit.Package()
+)
+
+// Config defines configuration struct for dbcleanup chore
+type Config struct {
+	SerialsInterval time.Duration `help:"how often to delete expired serial numbers" default:"24h"`
+}
+
+// Chore for deleting DB entries that are no longer needed.
+type Chore struct {
+	log    *zap.Logger
+	orders orders.DB
+
+	Serials sync2.Cycle
+}
+
+// NewChore creates new chore for deleting DB entries.
+func NewChore(log *zap.Logger, orders orders.DB, config Config) *Chore {
+	return &Chore{
+		log:    log,
+		orders: orders,
+
+		Serials: *sync2.NewCycle(config.SerialsInterval),
+	}
+}
+
+// Run starts the db cleanup chore
+func (chore *Chore) Run(ctx context.Context) (err error) {
+	defer mon.Task()(&ctx)(&err)
```

Comment:
We must remove this because it will create a very large span and it doesn't provide any important metric because this function should run from the satellite starts and until it ends.

Reply:
I don't see that much value on them.
On the other hand, @simongui reported having some performance issues with big traces.

In other new chores, we removed but we can discuss with whoever else has a better insight about it.

Reply:
I'm approving the PR despite this should be here or not.
Anyway, it isn't a big deal because many services are having it.